contrib/gocryptfs/template.py

```diff
@@ -0,0 +1,33 @@
+pkgname = "gocryptfs"
+pkgver = "2.4.0"
+pkgrel = 0
+hostmakedepends = [
+    "go",
+    "bash",
+    "pkgconf",
+]
+makedepends = [
+    "fuse-devel",
+    "openssl-devel",
+]
+pkgdesc = "Encrypted overlay filesystem"
+maintainer = "Nasado <hi@nasado.name>"
+license = "MIT"
+url = "https://github.com/rfjakob/gocryptfs"
+source = (
+    f"{url}/releases/download/v{pkgver}/{pkgname}_v{pkgver}_src-deps.tar.gz"
+)
+sha256 = "45158daf20df7f94e0c9ec57ba07af21df2e25e15b8584bf3c7de96adbbc2efd"
+# requires fuse kernel module, also wants the no-OpenSSL build to be static but we don't care about that
+# (no standard build system means no default check phase anyway, but still...)
+options = ["!check"]
+
+
+def do_build(self):
+    self.do("./build.bash")
```

Comment:
this doesn't seem to do anything interesting that the go build_style wouldn't do already (just pass the paths to build in make_build_args)

you can set main.GitVersion in there too via ldflags for the version

Reply:
it also sets GOAMD64=v2 which is wrong

Reply:
in case you don't feel like figuring it out:  

```
pkgname = "gocryptfs"
pkgver = "2.4.0"
pkgrel = 0
build_style = "go"
make_build_args = [
    "-ldflags="
    + f" -X main.GitVersion=v{pkgver}"
    + " -X main.GitVersionFuse=[vendored]",
    "./",
    "./gocryptfs-xray",
]
hostmakedepends = [
    "go",
    "pkgconf",
]
makedepends = ["openssl-devel"]
pkgdesc = "Encrypted overlay filesystem"
maintainer = "Nasado <hi@nasado.name>"
license = "MIT"
url = "https://github.com/rfjakob/gocryptfs"
source = (
    f"{url}/releases/download/v{pkgver}/{pkgname}_v{pkgver}_src-deps.tar.gz"
)
sha256 = "45158daf20df7f94e0c9ec57ba07af21df2e25e15b8584bf3c7de96adbbc2efd"
# requires fuse kernel module, also wants the no-OpenSSL build to be static but we don't care about that
# (no standard build system means no default check phase anyway, but still...)
options = ["!check"]


def post_install(self):
    self.install_license("LICENSE")
```

note fuse-devel is unused. also you probably want `depends = ["fuse"]`?

Reply:
you also need

```
    self.install_man("Documentation/gocryptfs.1")
    self.install_man("Documentation/gocryptfs-xray.1")
```

in post_install